### DISCUSSION TITLE
Units for ephys.stimuli?

### DISCUSSION THREAD

**Discussion by michmgee** [BlueBrain/BluePyOpt#480] [General]
I'm pretty new to this, so it might be something that is implied or that I don't know where in the documentation to look. What are the units for the step input amplitudes? 

I frequently see code that looks like this in the examples. When I see ('step1',10) is that 10 pA?

for protocol_name, amplitude in [('step1', 10)]:
        stim = ephys.stimuli.NrnSquarePulse(
                    step_amplitude=amplitude,
                    step_delay=100,
                    step_duration=500,
                    location=soma_loc,
                    total_duration=700)

Thanks very much!

**Comment by mgeplf** [comment]
They are in nA, per https://bluepyopt.readthedocs.io/en/latest/ephys/bluepyopt.ephys.stimuli.html#bluepyopt.ephys.stimuli.NrnSquarePulse

In python, you can access the documenation for a function by doing:
`help(ephys.stimuli.NrnSquarePulse)`, which should give:
```
>>> from bluepyopt.ephys.stimuli import NrnSquarePulse
>>> help(NrnSquarePulse)
[snip]
 |      Args:
 |          step_amplitude (float): amplitude (nA)
 |          step_delay (float): delay (ms)
 |          step_duration (float): duration (ms)
 |          total_duration (float): total duration (ms)
 |          location (Location): stimulus Location
[snip]
```
which should help in the future.

**Reply by michmgee** [reply]
Thanks for the tip!
